doc/media/openage/modpack_definition_file.md

```diff
@@ -0,0 +1,109 @@
+# Modpack Definition File
+
+The modpack definition file is the header file for [modpacks](modpacks.md).
+
+TOML is used as the configuration language.
+
+This document contains information about the parameters and sections that
+can be interpreted by the engine.
+
+## File Header
+
+The file starts with the comment:
+
+```
+# openage modpack definition file
```

Comment:
do we really want to enforce a comment?
i'd say this should be just convention.

Reply:
I'd say we should because people will then know what to search for when they want to change something.

Reply:
yes, but not enforce it, we should just generate it and recommend it in offered templates. the parser would be weird if we exclude all # lines except this exact one

Reply:
Rephrased as a recommendation + reasoning.